code/style.css

```diff
@@ -34,14 +34,14 @@ select {
   display: block;
   border: none;
   font-size: 22px;
```

Comment:
Looks like the text is a little too big for the box it's in, maybe go a couple of px smaller.